index.html

```diff
@@ -73,6 +76,237 @@ <h3 class="social_title blue">LET’S CONNECT</h3>
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <h3 class="social_title blue">LET’S CONNECT</h3>
+        <ul class="social_links">
+          <li class="social_link">
+            <a href="#"
+              ><img
+                class="social_icon"
+                src="./images/twitter.svg"
+                alt="Twitter"
+            /></a>
+          </li>
+          <li class="social_link">
+            <a href="#"
+              ><img
+                class="social_icon"
+                src="./images/Linkedin icon.svg"
+                alt="Linkedin"
+            /></a>
+          </li>
+          <li class="social_link">
+            <a href="#"
+              ><img class="social_icon M-icon" src="./images/Vector.svg" alt="M"
+            /></a>
+          </li>
+          <li class="social_link">
+            <a href="#"
+              ><img class="social_icon" src="./images/github.svg" alt="Github"
+            /></a>
+          </li>
+          <li class="social_link">
+            <a href="#"
+              ><img class="social_icon" src="./images/Vector (1).svg" alt="Diot"
+            /></a>
+          </li>
+        </ul>
+        <div class="action">
+          <button type="button" class="default_btn">Get my resume</button>
+        </div>
+        <ul class="about_ul">
+          <h3 class="sub_title">
+            Languages
+            <img
+              class="down_arrow_icon"
+              src="./images/Icon - Chevron - Down - Larger.png"
+              alt="down"
+            />
+          </h3>
+          <li class="ul_item">
+            <img src="./images/Ellipse 1.png" alt="Javascript" /><span
+              >Javascript</span
+            >
+          </li>
+          <li class="ul_item">
+            <img src="./images/Ellipse 1 (1).png" alt="HTML" /><span>HTML</span>
+          </li>
+          <li class="ul_item">
+            <img src="./images/Ellipse 1 (2).png" alt="CSS" /><span>CSS</span>
+          </li>
+        </ul>
+        <ul class="about_ul">
+          <h3 class="sub_title">
+            Frameworks
+            <img
+              class="arrow_icon"
+              src="./images/Icon - Chevron - Right - Larger.png"
+              alt="down"
+            />
+          </h3>
+        </ul>
+        <ul class="about_ul">
+          <h3 class="sub_title">
+            Skills
+            <img
+              class="arrow_icon"
+              src="./images/Icon - Chevron - Right - Larger.png"
+              alt="down"
+            />
+          </h3>
+        </ul>
+      </section>
+      <section class="contact_form">
+        <h2 class="title">Contact me</h2>
+        <p class="contact_form_des">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
```

Comment:
- [ ] As the previous code reviewer mentioned, please work on the style of your form heading and paragraph in order to match with the Figma template design. Kindly center the text, add some margin to the top and on the sides, and add the font family used in the whole template.
 -Your design:
<img width="478" alt="image" src="https://user-images.githubusercontent.com/61919617/200576160-372c4664-b4d8-4b99-8eee-41576efd1c85.png">